src/templates/header-two-tabs/style.scss

```diff
@@ -48,19 +50,21 @@
     border-left-width: 0;
 
     border-style: solid;
-    border-color: rgba(0, 0, 0, 0);
+    border-color: $black;
```

Comment:
This needs to be set to rgba(0,0,0,0) or else we get this rendering problem:

![screen shot 2016-03-18 at 2 42 38 pm](https://cloud.githubusercontent.com/assets/1119821/13888418/bb0ee7ee-ed17-11e5-81ec-36925ec7bb1f.png)

`border-color: transparent` is working well for me in the inspector.